ginga/misc/plugins/Histogram.py

```diff
@@ -441,10 +444,26 @@ def cut_levels(self):
         try:
             loval = float(self.w.cut_low.get_text())
             hival = float(self.w.cut_high.get_text())
+        except Exception as e:
+            errmsg = 'Error cutting levels: {0}'.format(str(e))
+            self.fv.showStatus(errmsg)
+            self.logger.error(errmsg)
+            return
 
+        # This is needed to avoid UserWarning from matplotlib
+        if loval == hival:
```

Comment:
Comparing just the strings won't work if I enter "0" for low and "0.0" for high.